packages/cardano-services/test/Reward/DbSyncRewardsProvider/__snapshots__/RewardsBuilder.test.ts.snap

```diff
@@ -8,16 +8,62 @@ Object {
 
 exports[`RewardsBuilder getAccountBalance returns AccountBalanceModel 1`] = `
 Object {
-  "balance": "48642468465",
+  "balance": "97675046319",
 }
 `;
 
 exports[`RewardsBuilder getRewardsHistory returns RewardEpochModel when there is epochs field 1`] = `Array []`;
 
-exports[`RewardsBuilder getRewardsHistory returns RewardEpochModel when there is no epochs field 1`] = `Array []`;
+exports[`RewardsBuilder getRewardsHistory returns RewardEpochModel when there is no epochs field 1`] = `
```

Comment:
This snapshot will fail in the next epoch. The result we snapshot needs to be consistent